src/Core/System/Snippet/SnippetService.php

```diff
@@ -314,12 +323,23 @@ private function getSnippetFilesByIso(array $isoList): array
     {
         $result = [];
         foreach ($isoList as $iso) {
-            $mappedIso = match ($iso) {
-                'de-DE' => 'de',
-                'en-GB' => 'en',
-                default => $iso,
-            };
-            $result[$iso] = $this->snippetFileCollection->getSnippetFilesByIso($mappedIso);
+            // Load all snippet files that match the exact locale (e.g., "es-AR")
+            $files = $this->snippetFileCollection->getSnippetFilesByIso($iso);
+            preg_match(
+                SnippetPatterns::COMPLETE_LOCALE_PATTERN,
+                $iso,
+                $matchedPattern,
+                \PREG_UNMATCHED_AS_NULL
+            );
+
+            // If the locale has a region (e.g., "es-AR"), try to load its base language ("es")
+            if (!empty($matchedPattern['region']) && strtolower($matchedPattern['region']) !== $iso) {
+                $fallbackFiles = $this->snippetFileCollection->getSnippetFilesByIso($matchedPattern['language']);
```

Comment:
```suggestion
                \assert(!empty($matchedPattern['language']));
                \assert(!empty($matchedPattern['region']));
                $fallbackFiles = $this->snippetFileCollection->getSnippetFilesByIso($matchedPattern['language']);
```

Recommend fixing PhpStand like this :)